pxr/usd/usdUtils/dependencies.cpp

```diff
@@ -655,8 +655,28 @@ class _AssetLocalizer {
             auto &fileAnalyzer = destFilePathAndAnalyzer.second;
 
             if (!fileAnalyzer.GetLayer()) {
-                _fileCopyMap.emplace_back(fileAnalyzer.GetFilePath(),
-                                          destFilePath);
+                const std::string srcFilePath = fileAnalyzer.GetFilePath();
+                if (TfStringContains(srcFilePath, "<UDIM>")) {
```

Comment:
This is now the third codesite where we do UDIM substitution, though I'm not too concerned about logic duplication here because this is the simplest (and only such) glob-style subst, and it's clearly needed!  But the OCD coder in me asks if you could follow the pattern that hdSt and usdImaging do in declaring a const to avoid duplicating the string literal four times?  E.g.

`static const char UDIM_PATTERN[] = "<UDIM>";`